agi_plz_ortschaften_pub/build.gradle

```diff
@@ -70,6 +110,22 @@ task convertData(type: Db2Db, dependsOn: 'dbImport'){
 }
 
 
+task copyDataSogisPub(type: Db2Db, dependsOn: 'copyDataPub'){
```

Comment:
Diesen Task braucht es nicht. Der Datenfluss ist ja

1. Import in die edit-DB, ins Schema agi_plz_ortschaften
2. Datenumbau innerhalb der edit-DB, ins Publikationsschema agi_plz_ortschaften_pub
3. Kopieren der Daten aus der edit-DB aus dem Publikationsschema agi_plz_ortschaften_pub in die pub-DB ins Publikationsschema agi_plz_ortschaften_pub

Und irgendwo nebenbei findet vorübergehend noch der Import in die sogis-DB, ins Schema agi_plz_ortschaften, statt. Mehr nicht.